common/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde_json = "1.0"
 serde_with = "2.0.1"
 slog = { version = "2.5", features = [ "max_level_trace", "release_max_level_debug" ] }
 smf = "0.2"
-steno = "0.2"
+steno = { git = "https://github.com/oxidecomputer/steno", branch = "node-access" }
```

Comment:
Ideally, I'd use a new rev of steno instead before merging. Same elsewhere in this PR